<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PV Defect Mapper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWebEngineView" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Dataset"/>
    <addaction name="actionClose_Dataset"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Annotation"/>
    <addaction name="actionLoad_Annotation"/>
    <addaction name="actionSave_Annotation"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionModule_Temperatures"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Dataset">
   <property name="text">
    <string>Open Dataset...</string>
   </property>
  </action>
  <action name="actionClose_Dataset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Dataset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAnnotation_Editor">
   <property name="text">
    <string>Annotation Editor</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Annotation">
   <property name="text">
    <string>New Annotation...</string>
   </property>
  </action>
  <action name="actionSave_Annotation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Annotation</string>
   </property>
  </action>
  <action name="actionLoad_Annotation">
   <property name="text">
    <string>Load Annotation...</string>
   </property>
  </action>
  <action name="actionModule_Temperatures">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Module Temperatures</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
